src/middlewared/middlewared/etc_files/local/inadyn.conf

```diff
@@ -31,7 +31,7 @@ provider ${config['provider']} {
     checkip-path = "${config['checkip_path']}"
 % endif
 % if config['username']:
-    username = ${config['username']}
+    username = "${config['username']}"
```

Comment:
Can username have a quote in itself?

Reply:
I haven't been able to find a reference which indicates that so far.